lib/task-data/tasks/boot-livecd.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Boot LiveCD',
+    injectableName: 'Task.Os.Boot.LiveCD',
+    implementsTask: 'Task.Base.Os.Install',
+    options: {
+        profile: 'boot-livecd.ipxe',
+        completionUri: 'renasar-ansible.pub',
```

Comment:
it is implementation specific on the LiveCD.  In the LiveCD I created, I issue a wget to retrieve a 'completionUrl' of 'init-commands.sh' which is a template of more commands to run.